AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DisplayMapperAdded.py

```diff
@@ -129,52 +188,233 @@ def AtomEditorComponents_DisplayMapper_AddedToEntity():
         Report.result(Tests.creation_redo, display_mapper_entity.exists())
 
         # 5. Set LDR color Grading LUT asset.
-        display_mapper_asset_path = os.path.join("TestData", "test.lightingpreset.azasset")
+        display_mapper_asset_path = os.path.join("LookupTables", "LUT_Sepia.azasset")
         display_mapper_asset = Asset.find_asset_by_path(display_mapper_asset_path, False)
         display_mapper_component.set_component_property_value(
-            AtomComponentProperties.display_mapper("LDR color Grading LUT"), display_mapper_asset.id)
+            AtomComponentProperties.display_mapper('LDR color Grading LUT'), display_mapper_asset.id)
         Report.result(
             Tests.ldr_color_grading_lut,
             display_mapper_component.get_component_property_value(
-                AtomComponentProperties.display_mapper("LDR color Grading LUT")) == display_mapper_asset.id)
+                AtomComponentProperties.display_mapper('LDR color Grading LUT')) == display_mapper_asset.id)
+
+        for operation_type in DISPLAY_MAPPER_OPERATION_TYPE.keys():
+            # 6. Set the Display Mapper Operation Type enumerated in DISPLAY_MAPPER_OPERATION_TYPE for each type
+            # Type property cannot be set currently. as a workaround we are calling an ebus to set the operation type
+            # display_mapper_component.set_component_property_value(
+            #     AtomComponentProperties.display_mapper('Type'), DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            render.DisplayMapperComponentRequestBus(
+                bus.Broadcast, "SetDisplayMapperOperationType", DISPLAY_MAPPER_OPERATION_TYPE[operation_type])
+            general.idle_wait_frames(3)
+            set_type = render.DisplayMapperComponentRequestBus(bus.Broadcast, "GetDisplayMapperOperationType")
+            Report.info(f"DiplayMapperOperationType: {set_type}")
+
+            # 7. Set Enable LDR color grading LUT property True
+            display_mapper_component.set_component_property_value(
+                AtomComponentProperties.display_mapper('Enable LDR color grading LUT'), True)
+            Report.result(
+                Tests.enable_ldr_color_grading_lut,
+                display_mapper_component.get_component_property_value(
+                    AtomComponentProperties.display_mapper('Enable LDR color grading LUT')) is True)
+
+
+            # 8. Set Override Defaults to False
```

Comment:
this doesn't match the docstring `8. Set Override Defaults to False then True` can you make it match then add an extra comment for each step i.e.:
```
8. Set Override Defaults to False then True
# Set Override Defaults to False
code here

# Set Override Defaults to True
code here
```

Reply:
Alternatively you can just do `8. Set Override Defaults to False then True` and then have all steps underneath it with no additional comments

Reply:
updated